src/main/java/io/github/mzmine/parameters/parametertypes/selectors/RawDataFilePlaceholder.java

```
@@ -186,6 +186,11 @@ public boolean isContainsZeroIntensity() {
         "This class is only to be used in the RawDataFilesSelection and does not support the required operation.");
   }
 
+  @Override
+  public boolean isContainsEmptyScans() {
+    return false;
```

Maybe throw unsupported operation exception